module/surveySubmissions/helper.js

```diff
@@ -85,6 +86,32 @@ module.exports = class SurveySubmissionsHelper {
           );
         }
 
+        if(surveySubmissionsDocument[0].programId){
+
+          let programDocument = 
+          await programsHelper.list(
+              {
+                  _id: surveySubmissionsDocument[0].programId,
+              },
+              ["name","description"],
+          );
+  
+          programDocument = programDocument.data.data;
```

Comment:
@borkarsaish65  like above please do the error validations

Reply:
done